### PR TITLE
Fixes for CalibrateIrtDlg

### PR DESCRIPTION
**Pull request by kaipot**

### CODE REVIEW

**Comment by kaipot** [comment]
Made a few changes after the demo over zoom (table allows rows to be deleted but otherwise is readonly)

File: pwiz_tools/Skyline/SettingsUI/Irt/CalibrateIrtDlg.cs [hunk 1]
```diff
@@ -782,14 +822,21 @@ public void SetFixedPoints(int one, int two)
             comboMaxPeptide.SelectedIndex = two;
         }
         #endregion
+
+        private void gridViewCalibrate_CellFormatting(object sender, DataGridViewCellFormattingEventArgs e)
+        {
+            if (e.Value is double value && double.IsNaN(value))
+            {
+                e.Value = !double.IsNaN(value) ? value.ToString(Formats.RETENTION_TIME, CultureInfo.CurrentCulture) : string.Empty;
```
Comment (bspratt) [inline pwiz_tools/Skyline/SettingsUI/Irt/CalibrateIrtDlg.cs:830]: Isn't it the case that double.IsNaN will always be false here?

File: pwiz_tools/Skyline/SettingsUI/PeptideGridViewDriver.cs [hunk 2]
```diff
@@ -46,6 +46,7 @@ protected PeptideGridViewDriver(DataGridViewEx gridView, BindingSource bindingSo
         }
 
         protected bool AllowNegativeTime { get; set; }
+        protected bool AllowMissingTime { get; set; }
```
Comment (bspratt) [inline pwiz_tools/Skyline/SettingsUI/PeptideGridViewDriver.cs:49]: Maybe a comment explaining why this is here?

**Review by bspratt — CHANGES_REQUESTED** [review]
Just a couple of notes, seems good and you've already demo'd for us.

File: pwiz_tools/Skyline/SettingsUI/Irt/CalibrateIrtDlg.cs (hunk 1 above)
Comment (kaipot) [reply pwiz_tools/Skyline/SettingsUI/Irt/CalibrateIrtDlg.cs:831]: Good catch, fixed

File: pwiz_tools/Skyline/SettingsUI/PeptideGridViewDriver.cs (hunk 2 above)
Comment (kaipot) [reply pwiz_tools/Skyline/SettingsUI/PeptideGridViewDriver.cs:52]: Added comment

**Pull request merged**